internal/kokoro/continuous.sh

```diff
@@ -47,6 +47,7 @@ set -x
 
 # cd to project dir on Kokoro instance
 cd github/cloud-bigtable-cbt-cli
+git config --global --add safe.directory "$(pwd)/./.git"
```

Comment:
Can you use a relative path instead? (`.git`) 

I don't know too much about the security implications of this change, but I don't like the idea of using an environment variable to mark a directory as "safe", since it could be easily manipulated